mta/fixtures/stakeholder_group.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from conftest import generate_string
+from swagger_client.models.api_stakeholder_group import ApiStakeholderGroup
+
+
+@pytest.fixture(scope="session")
+def stakeholderGroups(create_api, get_api):
```

Comment:
Please follow the Python convention: lowercase letters for functions and vars, words are separated by underscore. The fixture's name should be: stakeholder_group